ci-operator/step-registry/openstack/deprovision/deletefips/openstack-deprovision-deletefips-ref.yaml

```diff
@@ -6,9 +6,6 @@ ref:
     requests:
       cpu: 1000m
       memory: 100Mi
-  env:
-    - name: OS_CLOUD
-      documentation: name of cloud to use from clouds.yaml
```

Comment:
leave this as is, and instead give it a default.

Reply:
if instead you do this: 
env: 
   - name: OS_CLOUD
    documentation: name of the cloud provider to use from clouds.yaml
    default: "opesntack"

You accomplish in  what you want to do with one single line addition. , the next change is unecessary, and the step stays general.